pkg/controller/storagecluster/cephcluster.go

```diff
@@ -184,8 +186,8 @@ func newCephCluster(sc *ocsv1.StorageCluster, cephImage string, nodeCount int, s
 			},
 			Mgr: cephv1.MgrSpec{
 				Modules: []cephv1.Module{
-					cephv1.Module{Name: "pg_autoscaler", Enabled: true},
-					cephv1.Module{Name: "balancer", Enabled: true},
+					{Name: "pg_autoscaler", Enabled: true},
+					{Name: "balancer", Enabled: true},
```

Comment:
These changes are unrelated and should be either removed or in their own commit.